Automattic/woocommerce-payments
# Added small separator line between converted currency and the next line item

Fixes Part 1 of #7504 

#### Changes proposed in this Pull Request
Added a small separator line between converted currency and the next line item.








#### Testing instructions





- Create a dispute using the card `4000000000002685`
- Make sure the store currency differs from the account currency to render the explicit currency conversion.
- Match it with screenshot given in the issue.


![image](https://github.com/Automattic/woocommerce-payments/assets/15019298/09af761b-c346-4f6c-b61d-f5922059f03b)


-------------------


**Post merge**
